### PR TITLE
Auto install dependencies

### LINKED ISSUE #15: Prompt for auto-updating dependencies
> Check the possibility to insert a prompt in the one line installer to automatically update all affected outdated user's dependencies. That would allow the one-line installer to do "everything" for less experienced users and prevent the forum from being flooded with error reports and questions.

### PR DESCRIPTION
**Pull request by IvayloPapazov**
close #15

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -104,7 +104,14 @@ install_deps_osx() {
 
     echo -e "\nInstalling dependencies ...\n"
     brew update
-    brew install openssl libsodium
+
+    if [[ $(brew info openssl --json | jq ".[].installed|length > 0") = false ]]; then
```
Comment (dincho) [inline install.sh]: The installer should not rely on `jq` being installed on the user systems, most of the times it's not

File: install.sh [hunk 2]
```diff
@@ -104,7 +104,14 @@ install_deps_osx() {
...
+        brew install openssl
```
Comment (dincho) [inline install.sh]: Let's use explicit version here: `openssl@1.1`

File: install.sh (hunk 1 above)
Comment (IvayloPapazov) [reply install.sh]: c6acf82

File: install.sh (hunk 2 above)
Comment (IvayloPapazov) [reply install.sh]: c6acf82

**Review by venimus — DISMISSED** [review]

**Review by dincho — APPROVED** [review]

**Review by venimus — APPROVED** [review]

**Pull request merged**